Content.Shared/Cluwne/CluwneBrainComponent.cs

```diff
@@ -0,0 +1,14 @@
+using Robust.Shared.Audio;
+using Content.Shared.Chat.Prototypes;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Shared.Cluwne;
+
+[RegisterComponent]
+[NetworkedComponent]
+public sealed class CluwneBrainComponent : Component
```

Comment:
> just have hascomp in the reagent effect type's handler thing presumably corpium also has a zombie check

Good thinking, will look into it later. I think this is how i will have to do it since interactions with food seem a bit tricky otherwise.

Reply:
I tied to give access but doesnt work. Is this because the comp is in shared and the system in server? do i need to also make a shared cluwnebrainsystem?

Reply:
this should be done now (might wanna check it over)

Reply:
Getting errors in ubuntu, cant figure out what is causing it. Seems fine when i test it.